### PR TITLE
add loading-page

### PR DESCRIPTION
**Pull request by viniciusreiss**
close criar page loading #41

### CODE REVIEW

File: src/containers/Loading/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
```
Comment (alexandrejuk) [inline src/containers/Loading/index.js]: PropTypes is not used.

File: src/containers/Loading/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import './index.css'
+import { Button } from 'antd';
```
Comment (alexandrejuk) [inline src/containers/Loading/index.js]: Button  is not used.

File: src/containers/Loading/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import { Spin, Icon } from 'antd';
```
Comment (alexandrejuk) [inline src/containers/Loading/index.js]: Icon is not used.

File: src/containers/Loading/index.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+
+
+class Loading extends Component {
+	
+	render() {
+		const { name, onStart } = this.props
```
Comment (alexandrejuk) [inline src/containers/Loading/index.js]: props is not used

File: src/containers/Loading/index.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+		return (
```
Comment (alexandrejuk) [inline src/containers/Loading/index.js]: I think that you should improve the indentation.

File: src/containers/Loading/index.stories.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { HashRouter } from 'react-router-dom'
```
Comment (alexandrejuk) [inline src/containers/Loading/index.stories.js]: HashRouter is not used.

File: src/containers/Loading/index.js (hunk 1 above)
Comment (viniciusreiss) [reply src/containers/Loading/index.js]: resolve

File: src/containers/Loading/index.js (hunk 3 above)
Comment (viniciusreiss) [reply src/containers/Loading/index.js]: resolve

File: src/containers/Loading/index.stories.js (hunk 6 above)
Comment (viniciusreiss) [reply src/containers/Loading/index.stories.js]: resolve

**Pull request merged**